lib/mappings/mainContainer.js

```diff
@@ -0,0 +1,24 @@
+import { selectSelectedWallet } from './selectors';
+import { selectAccount, selectWallet } from '../actions';
+
+function mapStateToProps(state, otherProps) {
+  const selectedWallet = selectSelectedWallet(state);
+  return {
+    selectedWallet,
+    ...otherProps,
+  };
+};
+
+function mapDispatchToProps(dispatch) {
+  return {
+    selectAccount: async (walletId, accountId) =>
+      dispatch(selectAccount(walletId, accountId)),
```

Comment:
You shouldn't need the async here since there's not an await inside this function (there is in selectAccount itself, but that already has the async in the containing function declaration). when `selectAccount` is called later on, the containing function will have an `async` and then you would have something like `const acct = await selectAccount`.